coderd/jwtutils/jws.go

```diff
@@ -38,7 +38,7 @@ type Claims interface {
 }
 
 const (
-	signingAlgo = jose.HS512
+	SigningAlgo = jose.HS512
```

Comment:
I exported this constant variable for the `jose.ParseSigned` function calls in `coderd/workspaceapps/apptest/apptest.go`. See the previous review comment.